doc/source/whatsnew/v1.4.0.rst

```diff
@@ -258,7 +258,7 @@ Groupby/resample/rolling
 ^^^^^^^^^^^^^^^^^^^^^^^^
 - Fixed bug in :meth:`SeriesGroupBy.apply` where passing an unrecognized string argument failed to raise ``TypeError`` when the underlying ``Series`` is empty (:issue:`42021`)
 - Bug in :meth:`Series.rolling.apply`, :meth:`DataFrame.rolling.apply`, :meth:`Series.expanding.apply` and :meth:`DataFrame.expanding.apply` with ``engine="numba"`` where ``*args`` were being cached with the user passed function (:issue:`42287`)
--
+- Fixed bug in :meth:`SeriesGroupBy.value_counts` when the DataFrame/Series you are grouping has one row (:issue:`42618`)
```

Comment:
Only Series, I think. Even if you start with a DataFrame/DataFrameGroupBy object and then subset, this method is only acting on a Series.

Reply:
Also, move to 1.3.1